frontend/src/components/tabs/RuntimeNodeDetailsV2.tsx

```diff
@@ -173,7 +179,10 @@ function TaskNodeDetail({ runId, element, execution, namespace }: TaskNodeDetail
             )}
             {!logsBannerMessage && (
               <div className={commonCss.pageOverflowHidden}>
-                <LogViewer logLines={(logsDetails || '').split('\n')} />
+                <LogViewer
+                  logLines={(logsDetails || '').split('\n')}
+                  data-testid={'logs-view-window'}
```

Comment:
NIT: `'logs-view-window'` can be a const as well because it is used in the test.